scripts/release/mule/deploy/deb/deploy.sh

```diff
@@ -31,6 +19,8 @@ then
     SNAPSHOT="$CHANNEL-$VERSION"
 fi
 
+CHANNEL=$("./scripts/release/mule/common/get_channel.sh" "$NETWORK")
+VERSION=${VERSION:-$(./scripts/compute_build_number.sh -f)}
```

Comment:
Should these CHANNEL and VERSION definitions go above the SNAPSHOT definition?

Reply:
Fixed, good catch!